policies/new_chapter_policy.md

```diff
@@ -0,0 +1,136 @@
+# PyLadies Policy Proposal: New PyLadies Chapter Policy
+
+| Metadata |                                            |
+| -------- | ------------------------------------------ |
+| PPP      | 1                                          |
+| Version  | 1.0                                        |
+| Title    | New PyLadies Chapter Policy                |
+| Authors  | Mariatta <mailto:mariatta@pyladies.com>    |
+| Status   | Draft                                      |
+
+                                
+# TL;DR:
+
+The policy should answer the following questions
+
+1. Why is the policy necessary
+2. What is the purpose of a PyLadies chapter? What does it mean to start a PyLadies chapter? 
+3. What are minimum responsibilities and expectation when someone requested a PyLadies chapter?
+4. What kind of support can new chapter leaders expect from Global PyLadies leadership team?
+5. What questions do we ask in our form?
+6. How do we review new chapter requests?
+7. For accountability, how do we document new chapter creations?
+
+Timeline: I'd like to have this new process in place by end of January 2020.
+
+# Details
+
+## 1. Why is this policy necessary?
+
+- We had many inactive chapters, that never took off since they were created
+
+- We find chapters with more than one organizers to be more successful, and therefore it
+  should be part of a minimum requirement.
+  
+- We don't have clear definition of what  or why one would want to start a chapter.
+  Generally, there is expectation for ongoing and regular community event, instead of
+  a one time workshop.
+ 
+- Each PyLadies chapter uses resources in many forms: they might have Pro meetup
+  account from The PSF, they may have website hosted, they might require funding from The PSF.
+
+- PyLadies admins received numerous inquiries from community members regarding local
+  chapters that turned out to be inactive, taking up additional resource/volunteer time
+  from Global leadership and administrator team to address these inquiries. 
+ 
+ - By better vetting new chapters, we hope to reduce unnecessary administrative work
+   down the road, and better focus our efforts in helping active chapters.
+
+## 2. What does it mean to start a PyLadies chapter?
+
+We should define the purpose of local PyLadies chapter, and why a chapter would be necessary.
+
+- help build and grow local Python community, through meetups, or sharing info through mailing lists, twitter, etc.
+
+What are the roles of a PyLadies chapter leader?
+
+- share news from global PyLadies leadership to their members
+
+- organize at least one meetup per year (?)
```

Comment:
- not sure how to phrase it.. I would say that the chapter should aim at having regular meet-ups, no less often than one every two months. The chapter lead is responsible for managing that that happens by making sure that at any given time there are enough active co-organizers.

Reply:
I agree that have "regularly" scheduled meetups should be a requirement, but I am hesitant to subscribe a cadence. Some meetups don't hold events during seasonal times that are busy for members (like holiday periods). I like the X events per year approach, which I feel gives more leeway. Maybe 4/year (averages out to 1 event per 3 months)?

Reply:
We could also state this as "organizes or supports co-organization of at least N meetups per year" which allows community members to organize events to support the minimum N.

Reply:
I'm going with at least 1 /year for now, as I think it is generally achievable without demanding too much from volunteers.